tests/unit/cartography/intel/aws/test_apigateway.py

```diff
@@ -23,10 +23,13 @@ def test_none_policy():
     assert (res) is None
 
 
+@patch("time.sleep")
```

Comment:
Not related to this PR but I accidentally made the unit tests run 1 second slower in the previous PR (https://github.com/cartography-cncf/cartography/pull/1846) so I figured I'd fix it quickly here